scripts/skills/actives/rf_hook_shield_skill.nut

```diff
@@ -157,4 +177,20 @@ this.rf_hook_shield_skill <- ::inherit("scripts/skills/skill", {
 			}
 		}
 	}
+
+	function onTurnStart()
+	{
+		this.m.IsSpent = false;
+	}
+
+	function onGetHitFactors( _skill, _targetTile, _tooltip )
+	{
+		if (_skill != this || this.m.IsSpent || !_targetTile.IsOccupiedByActor || _targetTile.getID() != this.m.LastTargetID)
```

Comment:
`this.m.LastTargetID` contains the id of an entity but you compare it with the id of a tile here